src/app/+lookup-by-id/lookup-by-id-routing.module.ts

```diff
@@ -0,0 +1,43 @@
+import { LookupGuard } from './lookup-guard';
+import { NgModule } from '@angular/core';
+import { RouterModule, UrlSegment } from '@angular/router';
+import { ObjectNotFoundComponent } from './objectnotfound/objectnotfound.component';
+import { hasValue } from '../shared/empty.util';
+
+@NgModule({
+  imports: [
+    RouterModule.forChild([
+      {
+        matcher: (url) => {
+          // The expected path is :idType/:id
+          const idType = url[0].path;
+          let id;
+          // Allow for handles that are delimited with a forward slash.
+          if (url.length === 3) {
+            id = url[1].path + '/' + url[2].path;
+          } else {
+            id = url[1].path;
+          }
```

Comment:
If you replace this if and else with 
```typescript
id = url
  .slice(1)
  .map((us: UrlSegment) => us.path)
  .join('/');
 ``` 
it will assume that everything after the first url segment is part of the identifier so it will also support identifiers that have multiple slashes that may be added later on. 

You'll need to change the `hasValue`s in the if below with `isNotEmpty`s though (`isNotEmpty` will also check for empty strings). Which is probably best anyway.

Reply:
Done.